tests/unit_tests/template.py

```diff
@@ -0,0 +1,834 @@
+import json
+import re
+import subprocess
+import os
+import sqlite3 as lite
+import copy
+
+from nose.tools import assert_in, assert_true, assert_equals
+
+direc = os.listdir('./')
+for item in direc:
+    if item.endswith('.sqlite') or item.endswith('.json'):
+        os.remove(os.path.join('./', item))
```

Comment:
I would use glob here.

Reply:
Also, should this cleanup happen before and after every test function?

Reply:
For now, I wanted to see what the sqlite files looked like for debugging purposes, just to see what happens. But in the future, yes.